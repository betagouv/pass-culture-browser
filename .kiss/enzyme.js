import React from 'react'
import { shallow } from 'enzyme'

import MyComponent from '../MyComponent'

describe('src | components | MyComponent', () => {
  describe('snapshot', () => {
    it('should match snapshot', () => {
      const props = { errors: {} }
      const wrapper = shallow(<MyComponent {...props} />)
      expect(wrapper).toBeDefined()
      expect(wrapper).toMatchSnapshot()
    })
  })
})
