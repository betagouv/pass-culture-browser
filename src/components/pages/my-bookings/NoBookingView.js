/* eslint
  react/jsx-one-expression-per-line: 0 */
import React from 'react'
import { Link } from 'react-router-dom'

const NoBookingView = () => (
  <div className="is-full-layout flex-rows flex-center items-center">
    <p className="mb12">
      <Link
        to="/decouverte"
        className="is-white-text border-all border-white px24 py6"
      >
        <span className="fs22 is-medium">Lancez-vous</span>
      </Link>
    </p>
    <p className="is-white-text text-center">
      <span className="is-block">Dès que vous aurez réservé une offre,</span>
      <span className="is-block">vous la retrouverez ici.</span>
    </p>
  </div>
)

export default NoBookingView
