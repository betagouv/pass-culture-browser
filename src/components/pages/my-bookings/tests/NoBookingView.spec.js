import React from 'react'
import { shallow } from 'enzyme'
import NoBookingView from '../NoBookingView'

describe('src | components | pages | my-bookings | NoBookingView', () => {
  describe('snapshot', () => {
    it('should match snapshot', () => {
      // when
      const wrapper = shallow(<NoBookingView />)

      // then
      expect(wrapper).toBeDefined()
      expect(wrapper).toMatchSnapshot()
    })
  })
})
