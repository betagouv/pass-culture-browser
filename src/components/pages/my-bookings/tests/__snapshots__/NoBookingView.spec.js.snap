// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src | components | pages | my-bookings | NoBookingView snapshot should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NoBookingView />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p
          className="mb12"
        >
          <Link
            className="is-white-text border-all border-white px24 py6"
            to="/decouverte"
          >
            <span
              className="fs22 is-medium"
            >
              Lancez-vous
            </span>
          </Link>
        </p>,
        <p
          className="is-white-text text-center"
        >
          <span
            className="is-block"
          >
            Dès que vous aurez réservé une offre,
          </span>
          <span
            className="is-block"
          >
            vous la retrouverez ici.
          </span>
        </p>,
      ],
      "className": "is-full-layout flex-rows flex-center items-center",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Link
            className="is-white-text border-all border-white px24 py6"
            to="/decouverte"
          >
            <span
              className="fs22 is-medium"
            >
              Lancez-vous
            </span>
          </Link>,
          "className": "mb12",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <span
              className="fs22 is-medium"
            >
              Lancez-vous
            </span>,
            "className": "is-white-text border-all border-white px24 py6",
            "to": "/decouverte",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Lancez-vous",
              "className": "fs22 is-medium",
            },
            "ref": null,
            "rendered": "Lancez-vous",
            "type": "span",
          },
          "type": [Function],
        },
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <span
              className="is-block"
            >
              Dès que vous aurez réservé une offre,
            </span>,
            <span
              className="is-block"
            >
              vous la retrouverez ici.
            </span>,
          ],
          "className": "is-white-text text-center",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Dès que vous aurez réservé une offre,",
              "className": "is-block",
            },
            "ref": null,
            "rendered": "Dès que vous aurez réservé une offre,",
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "vous la retrouverez ici.",
              "className": "is-block",
            },
            "ref": null,
            "rendered": "vous la retrouverez ici.",
            "type": "span",
          },
        ],
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p
            className="mb12"
          >
            <Link
              className="is-white-text border-all border-white px24 py6"
              to="/decouverte"
            >
              <span
                className="fs22 is-medium"
              >
                Lancez-vous
              </span>
            </Link>
          </p>,
          <p
            className="is-white-text text-center"
          >
            <span
              className="is-block"
            >
              Dès que vous aurez réservé une offre,
            </span>
            <span
              className="is-block"
            >
              vous la retrouverez ici.
            </span>
          </p>,
        ],
        "className": "is-full-layout flex-rows flex-center items-center",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              className="is-white-text border-all border-white px24 py6"
              to="/decouverte"
            >
              <span
                className="fs22 is-medium"
              >
                Lancez-vous
              </span>
            </Link>,
            "className": "mb12",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <span
                className="fs22 is-medium"
              >
                Lancez-vous
              </span>,
              "className": "is-white-text border-all border-white px24 py6",
              "to": "/decouverte",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Lancez-vous",
                "className": "fs22 is-medium",
              },
              "ref": null,
              "rendered": "Lancez-vous",
              "type": "span",
            },
            "type": [Function],
          },
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <span
                className="is-block"
              >
                Dès que vous aurez réservé une offre,
              </span>,
              <span
                className="is-block"
              >
                vous la retrouverez ici.
              </span>,
            ],
            "className": "is-white-text text-center",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Dès que vous aurez réservé une offre,",
                "className": "is-block",
              },
              "ref": null,
              "rendered": "Dès que vous aurez réservé une offre,",
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "vous la retrouverez ici.",
                "className": "is-block",
              },
              "ref": null,
              "rendered": "vous la retrouverez ici.",
              "type": "span",
            },
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
